client/src/Pages/Search.tsx

```diff
@@ -4,12 +4,13 @@ import { useLocation } from 'react-router-dom';
 
 function Search() {
   const location = useLocation();
-  const searchLocation = location.state;
+  const search : any = location.state;
 
   return (
     <div className="container">
       {useTranslation().t('search-greeting')}
```

Comment:
{useTranslation().t('search-greeting')}
this text must be inside html element to make styling easy 
may be <h1> {useTranslation().t('search-greeting')} </h1> better?

Reply:
this text we don't need it at all just to show us which page we stand on {useTranslation().t('search-greeting')}